src/core.js

```diff
@@ -138,19 +150,19 @@ const Editable = module.exports = class Editable {
   * Adds the Editable.JS API to the given target elements.
   *
   * @method enable
-  * @param { jQuery element | undefined } target editable root element(s)
+  * @param { HTMLElement | undefined } target editable root element(s)
   *    If no param is specified all editables marked as disabled are enabled.
   * @chainable
   */
-  enable ($elem, normalize) {
-    const body = this.win.document.body
-    $elem = $elem || $('.' + config.editableDisabledClass, body)
-
+  enable (target, normalize) {
+    const document = this.win.document
     const shouldSpellcheck = this.config.browserSpellcheck
-    $elem.each((i, el) => {
-      block.init(el, {normalize, shouldSpellcheck})
-      this.dispatcher.notify('init', el)
-    })
+
+    target = target ? [target] : document.querySelectorAll(`.${config.editableDisabledClass}`)
```

Comment:
We could use a small helper here which would also keep this working with jquery elements.
```js
function toElementArray (node) {
  return node.jquery ? node.toArray() : [node]
}
```

Reply:
Sounds reasonable! Will do 👍